# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aop/cross.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-03-19 15:26:46.000000000 +0100
+++ ./Makefile	2004-04-14 22:22:18.401845144 +0200
@@ -1,13 +1,14 @@
 
-LDFLAGS = -lncurses
+prefix = /usr

 aop: aop.c
+	$(CC) aop.c -lncurses -o aop
 
 install: aop
-	install -d /usr/local/bin
-	install -d /usr/local/share/aop
-	install -m 755 aop /usr/local/bin/aop
-	install -m 644 aop-level-*.txt /usr/local/share/aop/
+	install -d $(DESTDIR)$(prefix)/bin
+	install -d $(DESTDIR)$(prefix)/share/aop
+	install -m 755 aop $(DESTDIR)$(prefix)/bin/aop
+	install -m 644 aop-level-*.txt $(DESTDIR)$(prefix)/share/aop/
 
 clean:
 	rm -f aop core* *~
